Ex5/Graph.java

```diff
@@ -0,0 +1,99 @@
+import java.util.LinkedList;
+import java.util.List;
+
+public class Graph {
+    private List<GraphNode> nodesList;
+
+    public Graph(){
+        nodesList = new LinkedList<>();

```

Comment:
You could just initialize it during declaration and remove constructor, as it does not do much.